src/docx2jats/objectModel/Document.php

```diff
@@ -177,6 +177,33 @@ static function getElementStyling(string $constStyleType, string $id): ?string {
 		}
 	}
 
+	static function getBuiltinStyle(string $constStyleType, string $id, array $builtinStyles): ?string {
+		// Traverse the chain of styles to see if the named id style
+		// inherits from one of the sought-for built-in styles and
+		// return the one that matches.
+		if (self::$stylesXpath) {
+			do {
+				$element = self::$stylesXpath->query("/w:styles/w:style[@w:type='" . $constStyleType . "'][@w:styleId='" . $id . "']")[0];
+
+				$basedOn = self::$stylesXpath->query("w:basedOn", $element)[0];
+				$id = $basedOn ? $basedOn->getAttribute("w:val") : null;
+
+				$name = self::$stylesXpath->query("w:name", $element)[0];
+				$styleName = $name->getAttribute("w:val");
+
+				if (in_array($styleName, $builtinStyles)) return $styleName;
```

Comment:
Sorry for such a long delay. Can you also convert $styleName in lower case before comparing with a built-in styles? e.g.: 
```
if (in_array(strtolower($styleName), $builtinStyles)) {
```